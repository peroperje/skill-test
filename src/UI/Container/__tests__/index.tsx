import React from 'react';
import { create } from 'react-test-renderer';

import { Container } from '../index';

describe('Container', () => {
  it('Snapshot', () => {
    const component = create(
      <Container>
        <>Some children</>
      </Container>
    );
    expect(component.toJSON()).toMatchSnapshot();
  });
});
