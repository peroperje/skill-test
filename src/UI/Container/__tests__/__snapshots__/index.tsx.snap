// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Container Snapshot 1`] = `
<div
  className="MuiContainer-root MuiContainer-maxWidthLg"
>
  Some children
</div>
`;
